Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/OnSearch.hs

```diff
@@ -154,6 +168,57 @@ mkQuoteEntities start end it = do
       item
     }
 
+mkQuoteEntitiesSpecialZone :: OS.StartInfo -> OS.StopInfo -> DSearch.SpecialZoneQuoteInfo -> QuoteEntities
+mkQuoteEntitiesSpecialZone start end it = do
+  let variant = castVariant it.vehicleVariant
+      estimatedFare = OS.DecimalValue $ toRational it.estimatedFare
+      fulfillment =
+        OS.FulfillmentInfo
+          { start,
+            end = Just end,
+            id = "fulf_" <> show it.quoteId,
```

Comment:
what is `fulf`

Reply:
This pattern we are already following while sending the fulfilment id.
I used same patten.